### ISSUE TITLE
What to do with `utop`

### ISSUE THREAD

**Issue by panglesd** [tarides/ocaml-platform-installer#30]
Including an `utop` binary does not allow to do `dune utop` which makes it loses some of its advantages.

What should we do about it? There are several possibilities:
- Remove it from the packaged installed by `ocaml-platform`
- Install it (solving the problem of dynamic linking one way or another), and explain that it should be replaced by the `opam install utop` to be able to run `dune utop`
- Install it via `opam install utop` (but this will install other dependencies as well, and seems not that good...).

Anyway, everything should changes when (if) we have a relocatable compiler. We can rethink the solution adopted at this point.

I might tend toward a pure removal of `utop` in the packaged installed by `ocaml-platform`, but I am not sure!

**Comment by kaddkaka** [comment]
So, `utop` is removed from the installation currently? Is that why I can't find it after doing the `ocaml-platform` step in this guide?

**Comment by panglesd** [comment]
Yes, `utop` is not installed by `ocaml-platform` currently. No libraries are installed by the installer, and `utop`'s libraries are required for `dune utop` to work.

To install `utop`, just run `opam install utop`.
